changes/1093.bugfix.md

```diff
@@ -0,0 +1 @@
+Take the major param for webhook without token endpoints when handling bucket ratelimits.
```

Comment:
I just realized, filename here is wrong

Reply:
Because the original pr was automatically closed when i force pushed an empty diff to avoid anybody else commenting on a mistake i was already in the middle of fixing